examples/script/core/1_basics.py

```diff
@@ -106,9 +106,9 @@
 
 # %% [markdown]
 """
-An `actor` is an object that processes user
+A `Pipeline` is an object that processes user
 inputs and returns responses.
-To create the actor you need to pass the script (`toy_script`),
+To create the pipeline you need to pass the script (`toy_script`),
 initial node (`start_label`) and
 the node to which the actor will default
```

Comment:
This is the first time actor is mentioned in `examples/script/core`. It should be explained or the mention should be removed.